frontend/src/homePage/LoginButton.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./LoginButton.css";
+
+class LoginButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="buttonBox">
+        <button
+          className="MyButton"
+          onClick={this.props.clickFunction}
+          style={{
+            backgroundColor: this.props.color,
+            fontSize: this.props.fontSize,
```

Comment:
again, consider using css classes rather than inline styles, especially if the style rules grows much bigger than this...